### DISCUSSION TITLE
Documentation for "Create one or more env vars" is buggy

### DISCUSSION THREAD

**Discussion by chandru89new** [vercel/vercel#8806] [General] [answered]
https://vercel.com/docs/rest-api#endpoints/projects/create-one-or-more-environment-variables

The text says you pass in `key, value, target...` but the code/spec shows nothing of the sort.

**Comment by amyegan** [comment]
Please visit https://github.com/vercel/vercel/discussions/8807 for further discussion

**Comment by amyegan** [comment, marked as answer]
The API docs recently got an update! They now include more detail about endpoint parameters and example values 🎉
